### PR TITLE
Error handling scraping server

### PR DESCRIPTION
**Pull request by kamalkish0r**
## Describe your changes and upload screenshots if changes on frontend/UI.
 - Added error handling in scraping server
 - Fixed issues occurred while scraping Myntra products
 - Added print statements to visualise tracking and errors, will remove before production phase
## Issue ticket number and link

Issue Fixes : #65 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Review by kamalkish0r — COMMENTED** [review]
Revisited!

File: servers/scrapping-server/app/product/routes.py [hunk 1]
```diff
@@ -7,28 +7,45 @@
...
-        'Currency': currency,
-        'MRP': mrp,
-        'Availability': availability,
-        'Rating': rating,
-        'Rating_Count': rating_count,
-        'Image_Link': img_link,
-    } 
-    '''
+    """
+    This function is a Flask route that takes a POST request with a product URL in the body, scrapes the
```
Comment (kanha638) [inline servers/scrapping-server/app/product/routes.py:11]: Please insert better comment string as it does not describe the route behaviours and parameters

**Review by kanha638 — CHANGES_REQUESTED** [review]
pease change these

File: servers/scrapping-server/app/product/routes.py (hunk 1 above)
Comment (kanha638) [reply servers/scrapping-server/app/product/routes.py:11]: i.e it should have request and response parameters as well not only definitions.

Comment (kamalkish0r) [reply servers/scrapping-server/app/product/routes.py:11]: it's mentioned that "it takes a POST request with a product URL in the body"

Comment (kanha638) [reply servers/scrapping-server/app/product/routes.py:11]: but it should also mention what should be the key value whether it should be `url` or `product_url` and same applies for the response as from public server we have to know about the whole pattern/format of request as well as response.

Comment (kamalkish0r) [reply servers/scrapping-server/app/product/routes.py:11]: Fixed,

**Review by kamalkish0r — COMMENTED** [review]
Fixed issue with doc string in add_product route

**Review by kanha638 — APPROVED** [review]
LGTM 👍🏻 . can be merged.

**Pull request merged**